less/index.less

```diff
@@ -47,13 +48,144 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
+/* Reset End */
 
+/* Less Start */
 /* Set every element's box-sizing to border-box */
+
+@batblue: rgb(10,145,191);
+@batdark: rgb(15,55,79);
+
+html { font-size: 62.5%; }
+
 * {
-    box-sizing: border-box;
-}
+  box-sizing: border-box;
+  font-size: 1.6rem;
+} 
 
 html, body {
     height: 100%;
 }
 
+body {
+	max-width: 880px;
+	height: 100%;
+	margin: 0 auto;
+	background-color: @batdark;
+
+	.container {
+		background-color: @batblue;
+		padding-left: 20px; 
+		padding-right: 20px;
+	}
+}
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 20px;
+	margin-bottom: 40px;
+	height: 100%;
+	width: 75%;
+	
+	a {
```

Comment:
perfect job demonstrating your understanding of how to nest elements and media queries. You nailed it.